app/components/CollectionIcon.js

```diff
@@ -14,12 +13,10 @@ type Props = {
 };
 
 function ResolvedCollectionIcon({ collection, expanded, size }: Props) {
-  const { ui } = useStores();
-
   // If the chosen icon color is very dark then we invert it in dark mode
   // otherwise it will be impossible to see against the dark background.
   const color =
-    ui.resolvedTheme === "dark" && collection.color !== "currentColor"
```

Comment:
Is it necessary to only have a color correction in dark mode?

Reply:
There is an assumption here that light mode is the default I suppose, but I don't see a reason to remove this check in this PR

Reply:
If the collection icon has a default color, it remains black when selecting it in the light mode.
That's the reason for the update. Is there a better way to do this?

Reply:
Typically we've just target the svg in css with an override, a better approach might be to add a color prop to CollectionIcon that we can use to pass white in when selected